src/api/@types/@enums.ts

```diff
@@ -3,3 +3,8 @@ export enum Status {
   COMPLETED = 'COMPLETED',
   NOT_STARTED = 'NOT_STARTED',
 }
+
+export enum Order {
+  RECENTLY = 'RECENTLY',
+  PREVIOUSLY = 'PREVIOUSLY',
+}
```

Comment:
enum 보다는 객체 상수 선언해서 쓰는 것을 추천드립니다.
https://engineering.linecorp.com/ko/blog/typescript-enum-tree-shaking
```suggestion
export const Order = {
  RECENTLY: 'RECENTLY',
  PREVIOUSLY: 'PREVIOUSLY',
} as const
```